tests/Core/Content/Product/SalesChannel/Listing/ProductSearchRouteTest.php

```diff
@@ -175,4 +178,46 @@ public function testSearchingVariantProductNumberWillShowVariant(
         $searchedProduct = $result->first();
         $this->assertSame($expectedProductNumber, $searchedProduct->getProductNumber());
     }
+
+    public function testAddElasticSearchContextForShopware64AndGreater(): void
```

Comment:
Maybe rephrase this a bit
```suggestion
    public function testElasticSearchContextIsAddedForShopware64AndGreater(): void
```